src/server_manager/web_app/ui_components/outline-intro-step.js

```diff
@@ -207,7 +207,8 @@ Polymer({
 
         <div id="gcp" class="card" on-tap="setUpGcpTapped">
           <div class="card-header">
-            <div class="tag">[[localize('setup-advanced')]]</div>
+            <div class="tag" hidden\$="{{isGcpAccountConnected}}">[[localize('setup-advanced')]]</div>
```

Comment:
Use `[[ ]]` instead

Reply:
Thanks. I changed the rest of the two-way bindings to one-way too.